packages/front-end/pages/experiments/index.tsx

```diff
@@ -327,13 +350,33 @@ const ExperimentsPage = (): React.ReactElement => {
   }
 
   const addExperimentDropdownButton = (
-    <DropdownMenu trigger="Add Experiment" menuPlacement="end">
-      <DropdownMenuItem onClick={() => setOpenNewExperimentModal(true)}>
-        Create New Experiment
-      </DropdownMenuItem>
-      <DropdownMenuItem onClick={() => setOpenImportExperimentModal(true)}>
-        Import Existing Experiment
-      </DropdownMenuItem>
+    <DropdownMenu
+      trigger={
+        <Button icon={<PiCaretDown />} iconPosition="right">
+          <PiPlusBold />
+        </Button>
+      }
+      menuPlacement="end"
+      tooltipContent="Add new..."
+    >
+      {canAddExperiment && (
+        <DropdownMenuItem onClick={() => setOpenNewExperimentModal(true)}>
+          Create New Experiment
+        </DropdownMenuItem>
+      )}
+      {canAddTemplate && (
+        <DropdownMenuItem onClick={() => setOpenTemplateModal({})}>
+          Create Template
+        </DropdownMenuItem>
+      )}
```

Comment:
Should we disable or hide this option if the org doesn't have the premium feature?

Reply:
Do we have a pattern for what we do with premium feature actions that show up in a dropdown? Do we just hide them or do we disable them and indicate they're premium somehow

Reply:
usually disable and wrap in a PremiumTooltip